eslintrc.json

```diff
@@ -1,21 +1,9 @@
 {
-  "extends": "standard",
+  "extends": "airbnb",
   "rules": {
-    "arrow-body-style": 2,
     "arrow-parens": [2, "as-needed"],
-    "camelcase": 2,
-    "default-case": 2,
-    "dot-notation": [2, { "allowPattern": "^[a-z]+(_+[a-z]+)+$"} ],
-    "no-else-return": 2,
-    "no-unused-vars": [2, { "vars": "all", "args": "after-used" }],
     "no-use-before-define": [2, "nofunc"],
```

Comment:
[airbnb enables this rule globally](https://github.com/airbnb/javascript/blob/396e4611c42416aa3b2af6ed193c3684251d9f68/packages/eslint-config-airbnb/rules/variables.js#L24). Using `"nofunc"` for us so that functions may be declared below their usage